lib/pipelines/__tests__/compileSoy.js

```diff
@@ -0,0 +1,411 @@
+'use strict';
+
+const assert = require('assert');
+const childProcess = require('child_process');
+const compileSoy = require('../compileSoy');
+const del = require('del');
+const ignore = require('gulp-ignore');
+const path = require('path');
+const sinon = require('sinon');
+const vfs = require('vinyl-fs');
+
+const outputDir = path.join(__dirname, '../../tmp');
+
+describe('Compile Soy Pipeline', function() {
+	afterAll(function(done) {
+		del(outputDir).then(function() {
+			done();
+		});
+	});
+
+	it('should compile soy files to js', function(done) {
+		const stream = vfs
+			.src('test/fixtures/soy/simple.soy')
+			.pipe(compileSoy());
+		stream.on('data', function(file) {
+			assert.strictEqual('simple.soy.js', file.relative);
+			expect(file.contents.toString()).toMatchSnapshot();
```

Comment:
Rather than always calling `toString()` in `toMatchSnapshot` we can add a [serializer](http://facebook.github.io/jest/docs/en/expect.html#expectaddsnapshotserializerserializer) to make life a bit easier for files